circle.yml

```diff
@@ -39,11 +39,8 @@ machine:
 
 checkout:
   post:
-    - |
-      git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
-      cd /home/ubuntu/st2/st2-packages
-      git checkout ${CIRCLE_BRANCH} || true
```

Comment:
For example when branch `v1.4` in `st2` is created and changes are pushed,
CI here will try to fetch `v1.4` branch of `st2-packages` which is not yet created.

It means that first build for new major release will be always `RED`.


Also interesting coincidence that you was mentioned there: https://github.com/StackStorm/st2/pull/2569#discussion_r55529678 to clarify existing release machinery :)